master/internal/job/job.go

```diff
@@ -87,8 +87,12 @@ func (j *Jobs) getJobs(
 		return nil, err
 	}
 
+	nonDaiJobs, _ := j.rm.GetNonDaiJobs(ctx, sproto.GetNonDaiJobs{
+		ResourcePool: resourcePool,
+	})
+
 	// Merge the results.
-	jobsInRM := make([]*jobv1.Job, 0, len(jobQ))
+	jobsInRM := make([]*jobv1.Job, 0, len(jobQ)+len(nonDaiJobs))
```

Comment:
I'm not sure how you are intending to represent non Determined jobs.   
If you are just going to return an existing jobv1.Job -- why do you need a new method?
If it really is just the same type, why wouldn't we just return them all in the same call?

Reply:
Got it.   I was thinking that GetJobs was already calling the RM instead of using local job.go data, so I thought you could augment it in the existing interface instead of appending it here.

So this makes sense now.